sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2196,6 +2196,29 @@ object SQLConf {
       .checkValue(bit => bit >= 10 && bit <= 30, "The bit value must be in [10, 30].")
       .createWithDefault(16)
 
+  val SKIP_PARTIAL_AGGREGATE_ENABLED =
+    buildConf("spark.sql.aggregate.partialaggregate.skip.enabled")
+      .internal()
+      .doc("Avoid sorter(sort/spill) during partial aggregation")
+      .booleanConf
+      .createWithDefault(true)
+
+  val SKIP_PARTIAL_AGGREGATE_THRESHOLD =
+    buildConf("spark.sql.aggregate.partialaggregate.skip.threshold")
```

Comment:
I think we need to assign more meaningful param names, e.g.,
```

spark.sql.aggregate.skipPartialAggregate
spark.sql.aggregate.skipPartialAggregate.minNumRows
spark.sql.aggregate.skipPartialAggregate.aggregateRatio
```

Reply:
Done